go/vt/vtctl/vtctl.go

```diff
@@ -3240,37 +3241,41 @@ func commandApplySchema(ctx context.Context, wr *wrangler.Wrangler, subFlags *fl
 	if err != nil {
 		return err
 	}
-	executionUUID, err := schema.CreateUUID()
-	if err != nil {
-		return err
+
+	var cID *vtrpcpb.CallerID
+
+	if *callerID != "" {
+		cID = &vtrpcpb.CallerID{Principal: *callerID}
 	}
+
 	if *migrationContext == "" {
 		// -request_context is a legacy flag name
 		// we now prefer to use -migration_context, but we also keep backwards compatibility
 		*migrationContext = *requestContext
 	}
-	if *migrationContext == "" {
-		*migrationContext = fmt.Sprintf("vtctl:%s", executionUUID)
-	}
-	executor := schemamanager.NewTabletExecutor(*migrationContext, wr, *waitReplicasTimeout)
-	if *allowLongUnavailability {
-		executor.AllowBigSchemaChange()
-	}
-	if *skipPreflight {
-		executor.SkipPreflight()
-	}
-	if err := executor.SetDDLStrategy(*ddlStrategy); err != nil {
+
+	log.Info("Calling ApplySchema on VtctldServer")
+	resp, err := wr.VtctldServer().ApplySchema(ctx, &vtctldatapb.ApplySchemaRequest{
+		Keyspace:                keyspace,
+		AllowLongUnavailability: *allowLongUnavailability,
+		DdlStrategy:             *ddlStrategy,
+		Sql:                     strings.Split(change, ","),
+		SkipPreflight:           *skipPreflight,
+		UuidList:                strings.Split(*uuidList, ","),
```

Comment:
```suggestion
		UuidList:                textutil.SplitDelimitedList(*uuidList),
```

`textutil.SplitDelimitedList()` also safely ignores spaces or empty entries